engine/src/main/java/org/terasology/rendering/nui/widgets/UIList.java

```diff
@@ -39,7 +39,7 @@
 
     private Binding<Boolean> selectable = new DefaultBinding<>(true);
     private Binding<T> selection = new DefaultBinding<>();
-    private Binding<List<T>> list = new DefaultBinding<List<T>>(Lists.<T>newArrayList());
+    private Binding<List<T>> list = new DefaultBinding<>(Lists.<T>newArrayList());
```

Comment:
Using `new ArrayList<>()` should be unambiguous and work even without the type specifier.